### ISSUE TITLE
sigil-toolkit is private

### ISSUE THREAD

**Issue by nmushegian** [urbit/sigil-js#20]
='[

https://github.com/urbit/sigil-toolkit

**Comment by galenwp** [comment]
Yeah, I hear you. 

Gonna leave it private for now — they're just the tools we used to build sigils. Someday we'd like to open it up, and write some about the process of designing these things.

**Comment by nmushegian** [comment]
There's links to it in this repo, don't tease us!

**Comment by galenwp** [comment]
Ha, fair enough! I was wondering where you got the link...

@gavinpatkinson can we take those out?

**Comment by johnmendonca** [comment]
I can understand leaving this project private.

It would be helpful if there were a sample web project that uses this one as a dependency.  I have spent some time trying to get this to work "properly" but have been unable to.

I am able to generate sigils now, but only by modifying the test script in this repo :smile: 

I tried running node on the build output, and also included this project as a git dependency on another project.  In both cases it had issues with the new import/export keywords.  I am using node v10.8 and this project builds using babel so I wouldn't expect this.  Sorry that I am not up to date on the latest js build practices.

Perhaps the code running at http://sigil.tlon.io/ would make a good example?  Thanks, I have been quite interested in this lately.
